modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,97 @@
+
+* With this release, logging no longer supports the `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` custom resources. Refer to the product documentation for details on the replacement features. (link:https://issues.redhat.com/browse/LOG-5803[LOG-5803])
+
+* With this release,logging no longer manages or deploys log storage (such as Elasticsearch), visualization (such as Kibana), or Fluentd-based log collectors. (link:https://issues.redhat.com/browse/LOG-5368[LOG-5368])
+
+[NOTE]
+====
+In order to continue to use Elasticsearch and Kibana managed by the elasticsearch-operator, the administrator must modify those object's ownerRefs before deleting the ClusterLogging resource.
+====
+
+[id="logging-release-notes-6-0-0-enhancements"]
+== New features and enhancements
+=== Log Collection
+
+* This feature introduces a new architecture for Red Hat OpenShift Logging by shifting component responsibilities to their relevant operators, such as for storage, visualization, and collection. It introduces the `ClusterLogForwarder.observability.openshift.io` API for log collection and forwarding. Support for the `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` APIs, along with the Red Hat managed Elastic stack (Elasticsearch and Kibana), is removed. Users are encouraged to migrate to the Red Hat `LokiStack` for log storage. Existing managed Elasticsearch deployments can be used for a limited time. Automated migration for log collection is not provided, so administrators need to create a new ClusterLogForwarder.observability.openshift.io specification to replace their previous custom resources. Refer to the official product documentation for more details. (link:https://issues.redhat.com/browse/LOG-3493[LOG-3493])
+
+* Before this update, the Vector collector deployments had no limits on memory and CPU usage. With this update, default requests and limits are set based on the recommendations from the Vector documentation. (link:https://issues.redhat.com/browse/LOG-4745[LOG-4745])
+
+* This enhancement brings Vector in line with the upstream Vector version v0.37.1. (link:https://issues.redhat.com/browse/LOG-5296[LOG-5296])
+
+* With this release, an alert triggers when log collectors buffer logs to a node's file system and use over 15% of the available space. This may indicate the log collectors are facing back pressure from their outputs, and administrators should act to prevent potential issues on the cluster node. (link:https://issues.redhat.com/browse/LOG-5381[LOG-5381])
+
+* This enhancement updates the selectors for all components to use common Kubernetes labels. (link:https://issues.redhat.com/browse/LOG-5906[LOG-5906])
+
+* With this update, the collector configuration will deploy as a ConfigMap instead of a Secret. This change allows users to view and edit the configuration when the ClusterLogForwarder is set to `Unmanaged`.(link:https://issues.redhat.com/browse/LOG-5599[LOG-5599])
```

Comment:
```suggestion
* With this update, the collector configuration deploys as a `ConfigMap` instead of a secret. This change allows users to view and edit the configuration when the `ClusterLogForwarder` is set to `Unmanaged`. (link:https://issues.redhat.com/browse/LOG-5599[LOG-5599])
```

Reply:
- Avoid future tense, so will deploy -> deploys, per IBMSG.
- Config map either needs to be two separate words or in backticks, see https://redhat-documentation.github.io/supplementary-style-guide/#config-map
- I don't think secrets needs to be capitalized. 
- I think ClusterLogForwarder needs backticks since its an API.